src/NClient/NClient.Common/Helpers/EnumExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.ComponentModel;
+
+namespace NClient.Common.Helpers
+{
+    // ReSharper disable once UnusedType.Global
+    public static class EnumExtensions
```

Comment:
This class should be internal, since its method extends all enums, this can annoy users. Also, it is not the responsibility of NClient

Reply:
ok